AuthWithIdentity/Controllers/AccountController.cs

```diff
@@ -117,17 +117,25 @@ public IActionResult Register(string returnUrl = null)
                     // Send an email with this link
                     var code = await _userManager.GenerateEmailConfirmationTokenAsync(user);
                     var callbackUrl = Url.Action(nameof(ConfirmEmail), "Account", new { userId = user.Id, code = code }, protocol: HttpContext.Request.Scheme);
-                    await _emailSender.SendEmailAsync(model.Email, "Confirm your account",
+                    try {
```

Comment:
Please put braces on separate lines, according to C# conventions.
Visual Studio should do this automatically for you. Or click Edit > Advanced > Format Document (Ctrl+E+D).